src/styles/app.css

```diff
@@ -0,0 +1,17 @@
+.app {
+  display: flex;
+  flex-direction: column;
+}
+
+.bg {
+  height: 100vh;
+  width: 100vw;
+  background-image: url("https://d2pn8kiwq2w21t.cloudfront.net/original_images/jpegPIA24571.jpg");
+  background-size: contain;
+}
```

Comment:
Theres a bit of a bug with how this looks, theres a large blank area below the background image. Theres a numerous ways to sort this out depending on what you want this to look like, but an example fix using a repeating pattern with the existing markup you have would be to remove the width and height properties (they wont be needed anymore) and adding the `overflow-y: scroll` property:

```
.bg {
  overflow-y: scroll;
  background-image: url("https://d2pn8kiwq2w21t.cloudfront.net/original_images/jpegPIA24571.jpg");
  background-size: contain;
}
```

At least gets rid of the glaring white background at the bottom, depending what you want to do theres loads of different approaches though